web/models/setting/setting.ex

```diff
@@ -0,0 +1,37 @@
+defmodule CoursePlanner.Setting do
+  @moduledoc """
+  This module holds the model for the setting table
+  """
+  use CoursePlanner.Web, :model
+
+  schema "settings" do
+    field :notification_frequency, :integer
```

Comment:
maybe the naming is bad. but It is suppose to be used for how often attendance notification should be send to teachers. the number suppose to be in days. (every 1 day, every 2 day and so on)

PS : I can additionally add a boolean for it to be sent at all or not

but if you think the naming is not good do tell, cause I had my mind froze on this